desktop/src/main/java/bisq/desktop/main/dao/economy/supply/dao/DaoChartDataModel.java

```diff
@@ -119,6 +136,16 @@ long getProofOfBurnAmount() {
         Map<Long, Long> compensationMap = getCompensationByInterval();
         Map<Long, Long> reimbursementMap = getReimbursementByInterval();
         totalIssuedByInterval = getMergedMap(compensationMap, reimbursementMap, Long::sum);
+
+        var items = totalIssuedByInterval
+                .entrySet()
+                .stream()
+                .sorted(Comparator.comparing(Map.Entry::getKey))
+                .collect(Collectors.toList());
+
+        items.forEach(i -> {
+            log.info("" + i.getKey());
```

Comment:
Is there a reason why you are logging this?

Reply:
Fixed

Reply:
Please don't force-push when in review, just make another commit. Otherwise it is harder to update the branch and to look at the changes.

Reply:
OK - my mistake with leaving this logging  code was so pitiful that I just wanted to clear it from git history :)